### ISSUE TITLE
Option to hide "Launch TensorBoard Session"

### ISSUE THREAD

**Issue by yzhang-gh** [microsoft/vscode-jupyter#10388]
The integration of TensorBoard is nice. However, it would be better if we can disable the code lens. It has been requested before but the solution doesn't work now as this feature is no longer in an experimental stage.

https://github.com/microsoft/vscode-python/issues/15664, https://github.com/microsoft/vscode-python/issues/14806#issuecomment-885786603

**Comment by IanMatthewHuff** [comment]
Thanks for the suggestion. Certainly something that we can do, just might discuss quick at triage if we should just have this respect our main setting regarding our debugging code lenses or have a new setting just for tensor board.

**Comment by yzhang-gh** [comment]
Hi all, I didn't remember why I opened the issue in this repo as it actually should be done in the vscode-python side. Could someone help me transfer this issue there?

**Comment by DonJayamanne** [comment]
@yzhang-gh Yes, ideally it should be in the Python extension, however our team (& this product) we own that component in the Python extension hence we'll be responsible for the changes.
Apologies for the complications.

**Comment by yzhang-gh** [comment]
I see. And could you give me some hints about which files I need to change to implement this? Will try to open a PR when I get some time.

**Comment by DonJayamanne** [comment]
This issue has been open  for a while and we haven't received any upvotes on this issue, hence closing for now
